src/postamble_minimal.js

```diff
@@ -77,10 +77,29 @@ WebAssembly.instantiate(Module['wasm'], imports).then(function(output) {
   var asm = output.instance.exports;
 #if DECLARE_ASM_MODULE_EXPORTS == 0
 
+#if WASM_BACKEND
+  // XXX Hack: some function names need to be mangled when exporting them from wasm module, others do not. 
+  // https://github.com/emscripten-core/emscripten/issues/10054
+  // Keep in sync with emscripten.py function treat_as_user_function(name).
```

Comment:
I think we can add the array of names here to `settings_internal.js`, which is available both in `emscripten.py` and here.